core_v2/sources/domains.move

```diff
@@ -700,25 +692,35 @@ module aptos_names_v2::domains {
     /// Entry function for clearing reverse lookup.
     public entry fun clear_reverse_lookup_entry(
         account: &signer
-    ) acquires NameRecordV2, ReverseLookupRegistryV1, SetReverseLookupEventsV1 {
+    ) acquires CollectionCapabilityV2, NameRecordV2, ReverseRecord, SetReverseLookupEventsV1 {
         clear_reverse_lookup(account);
     }
 
     /// Clears the user's reverse lookup.
-    public fun clear_reverse_lookup(account: &signer) acquires NameRecordV2, ReverseLookupRegistryV1, SetReverseLookupEventsV1 {
+    public fun clear_reverse_lookup(
+        account: &signer
+    ) acquires CollectionCapabilityV2, NameRecordV2, ReverseRecord, SetReverseLookupEventsV1 {
         let account_addr = signer::address_of(account);
         clear_reverse_lookup_internal(account_addr);
     }
 
+    inline fun reverse_record_addr(account_addr: address): address acquires CollectionCapabilityV2 {
+        // TODO: Use object ExtendRef
+        object::create_object_address(
+            &get_token_signer_address(),
+            bcs::to_bytes(&account_addr),
+
+        )
+    }
+
     /// Returns the reverse lookup (the token addr) for an address if any.
     public fun get_reverse_lookup(
```

Comment:
this returns the address of `NameRecordV2` token if exist, right?

Reply:
if i have a domain (a `NameRecordV2` token under my address 0x123), i transfer the domain to Brian, the token is under Brian's address 0x456 now. 

say someone wants to reverse lookup from my address 0x123, but now the token is under Brian, is it still possible to look it up?

Reply:
guess im wondering will `create_object_address` work if `account_addr` is different from the token or object owner?
```    inline fun reverse_record_addr(account_addr: address): address acquires CollectionCapabilityV2 {
        // TODO: Use object ExtendRef
        object::create_object_address(
            &get_token_signer_address(),
            bcs::to_bytes(&account_addr),

        )
    }
```

Reply:
i guess it still works, `create_object_address` will generate the token or object address deterministically